src/metabase/cmd/copy.clj

```diff
@@ -95,7 +95,9 @@
     :model/ModelIndex
     :model/ModelIndexValue
     ;; 48+
-    :model/TablePrivileges]
+    :model/TablePrivileges
+    :model/AuditLog
+    :model/RecentViews]
```

Comment:
Already added these [here](https://github.com/metabase/metabase/pull/34585) but I guess isn't not a merge conflict since the code is identical